### PR TITLE
[GNA] Add functional tests checking accuracy for pwl_me changes

### PR DESCRIPTION
**Pull request by marcinkusm**
### Details:
 - Added functional tests checking accuracy for pwl_me changes. If accuracy does not decreased when pwl_me is decreased.

### Tickets:
 - 95144

### CODE REVIEW

File: src/plugins/intel_gna/tests/functional/shared_tests_instances/pw_me_validation_test.cpp [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+                                                                     high_node_out,
+                                                                     levels16);
+
+        ngraph::ResultVector results{std::make_shared<ngraph::opset8::Result>(fq_out)};
+        function = std::make_shared<ngraph::Function>(results, params, "ActivityFq");
+    }
+
+    void Run() override {
+        // execute network for different pwl_me options
+        std::vector<std::string> pwl_mes = {"1.0", "0.6", "0.3", "0.25", "0.1", "0.05"};
```
Comment (marcinkusm) [inline src/plugins/intel_gna/tests/functional/shared_tests_instances/pw_me_validation_test.cpp:143]: change this to use step 0.1

File: src/plugins/intel_gna/tests/functional/shared_tests_instances/pw_me_validation_test.cpp [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+
+typedef std::tuple<InferenceEngine::Precision,       // Network Precision
+                   std::pair<float, float>,          // Input values
+                   ngraph::helpers::ActivationTypes  // Activation type
+                   >
+    eltwiseActFqParams;
+
+namespace LayerTestsDefinitions {
+
+class PWlMeTest : public testing::WithParamInterface<eltwiseActFqParams>, public LayerTestsUtils::LayerTestsCommon {
```
Comment (marcinkusm) [inline src/plugins/intel_gna/tests/functional/shared_tests_instances/pw_me_validation_test.cpp:57]: verify if proper base class is used

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**